vega_sim/service.py

```diff
@@ -2070,3 +2070,23 @@ def list_transfers(
             party_id=party_id,
             direction=direction,
         )
+
+    def get_liquidity_fee_shares(
+        self, market_id: str, wallet_name: str, key_name: Optional[str] = None
+    ) -> Union[Dict, float]:
```

Comment:
This should probably have wallet_name as default `None` too so that it's obvious you can just get with `market_id`